static/app/components/acl/access.tsx

```diff
@@ -52,9 +52,10 @@ export type Props = {
  */
 function Access({children, isSuperuser = false, access = [], team, project}: Props) {
   const config = useLegacyStore(ConfigStore);
-  const organization = useOrganization();
+  const organization = useContext(OrganizationContext);
 
-  const {access: orgAccess} = organization || {access: []};
+  const {access: orgAccess} =
+    organization || ({access: []} as {access: Organization['access']});
```

Comment:
I’m 👎 to this approach. It’s just masking the issue.

I’d prefer for us to figure out how to get organization context up to the modal.

Reply:
With this there’s no longer an error, but will just hide access to features. Making for tricky to track down bugs

Reply:
yes but this is a hot fix, so folks can use a few things from the command palette until we have time to figure out how to make `useOrganization` work in modals. But in any case, I think putting back `withOrganization` for now is a better approach and I have updated the code 😉